system/Commands/Database/ShowTableInfo.php

```diff
@@ -244,6 +267,11 @@ private function makeTbodyForShowAllTables(array $tables): array
         return $this->tbody;
     }
 
+    /**
+     * @used-by showDataOfTable
```

Comment:
```suggestion
     * @used-by showDataOfTable()
```
By the way, do we really need `@used-by` like this?

Reply:
Can you remove all `@used-by`?
I think we don't need `@used-by` when we can jump to callers in an IDE.